sn_networking/src/lib.rs

```diff
@@ -83,9 +84,9 @@ const IDENTIFY_CLIENT_VERSION_STR: &str = concat!("safe/client/", env!("CARGO_PK
 const IDENTIFY_PROTOCOL_STR: &str = concat!("safe/", env!("CARGO_PKG_VERSION"));
 
 /// Duration to wait for verification
-const VERIFICATION_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(3);
+const REVERIFICATION_WAIT_TIME_S: std::time::Duration = std::time::Duration::from_secs(3);
 /// Number of attempts to verify a record
-const VERIFICATION_ATTEMPTS: usize = 3;
+const VERIFICATION_ATTEMPTS: usize = 30;
```

Comment:
30? how comes requiring that high number?
given each iteration is 3s, it becomes 90s in total to complete?

Reply:
This should probs not be in this PR. Buttt, i saw `Ci failure without, and anselme was having a real bad time trying to verify uploads, so it needs to be higher. 

Could be 30s, or so. I'll open that up as a separate pr.